src/components/Footer/Footer.jsx

```diff
@@ -1,7 +1,11 @@
 import styles from "./Footer.module.css";
 
 function Footer() {
-  return <div className={styles.wrapper}>SiteFooter</div>;
+  return <div className={styles.wrapper}>
+    <p> <a href="#">Terms and Conditions</a></p>
```

Comment:
I am juzt thinking just because it is react maybe we can add the Link? 
for example:
<li>
         <Link to="#">Terms and Conditions</Link>
 </li>

Reply:
have added the link